pkg/kubelet/winstats/perfcounter_nodestats.go

```diff
@@ -0,0 +1,143 @@
+// +build windows
+
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package winstats
+
+import (
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+	"sync"
+
+	cadvisorapi "github.com/google/cadvisor/info/v1"
+)
+
+// PerfCounterNodeStatsClient is a client that provides Windows Stats via PerfCounters
+type PerfCounterNodeStatsClient struct {
+	nodeStats
+	mu sync.Mutex
+}
+
+// NewPerfCounterNodeStatsClient creates a new client
+func NewPerfCounterNodeStatsClient() *PerfCounterNodeStatsClient {
+	return &PerfCounterNodeStatsClient{}
+}
+
+func (p *PerfCounterNodeStatsClient) startMonitoring() error {
+	memory, err := getPhysicallyInstalledSystemMemoryBytes()
```

Comment:
Do we need to handle the error here?

Reply:
Thanks for pointing it out. I added a check for that error.